chef/cookbooks/cpe_chrome/resources/cpe_chrome.rb

```diff
@@ -27,7 +27,8 @@ def manage_chrome
   end
 
   def manage_chrome_osx
-    prefs = node['cpe_chrome'].reject { |_k, v| v.nil? }
+    prefs = node['cpe_chrome']['profile'].reject \
+      { |_k, v| v.nil? || v.class == Chef::Node::ImmutableArray && v.empty? }
```

Comment:
This does not work as expected as it does not reject nil values. I think there is a bit of misunderstanding on the code.